test/configuration-tests/with-networks/check.ts

```diff
@@ -43,3 +43,12 @@ process.env.NETWORK = invalidNetwork;
 execution = hardhatStarknetTest("--no-compile test/contract-factory-test.ts".split(" "), true);
 assertContains(execution.stderr, expected);
 console.log("Success");
+
+console.log("Testing with alpha-goerli2 config network");
+process.env.NETWORK = "alpha-goerli2";
+const network = ensureEnvVar("NETWORK");
```

Comment:
What's the purpose of first storing it as an env var and then loading it back to a JS variable?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/258/commits/2c7386c5c9ff8f72caefd39c824a0b9101136789